### PR TITLE
Add Group Search

### PR DESCRIPTION
**Pull request by PCOffline**

### CODE REVIEW

**Comment by PCOffline** [comment]
One problem is that the auto-complete now pushes the list down. I had to adjust the positions of the search-bar to not be absolute and that's a side-effect. Will work on that soon

**Comment by PCOffline** [comment]
Should there be an icon in the search bar of a group that the user cannot edit?

File: src/components/shared/users-input-fields/UsersInputFields.jsx [hunk 1]
```diff
@@ -79,11 +88,45 @@ const UsersInputFields = ({
...
+            timeout={500}
+          >
+            <AddIcon />
+          </Fade>
+        </IconButton>
+        <SearchBar
+          setSelectedUser={setSelectedUser}
+          setSearchedUsers={setSearchedUsers}
+          groupUsers={groupUsers}
+          className={classes.inputField}
```
Comment (carmeltsadok) [inline src/components/shared/users-input-fields/UsersInputFields.jsx:124]: It doesn't affect the component

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/components/shared/users-input-fields/UsersInputFields.jsx [hunk 2]
```diff
@@ -79,11 +89,48 @@ const UsersInputFields = ({
...
+            mountOnEnter
+            unmountOnExit
+            exit={false}
+            timeout={500}
+          >
+            <Tooltip title={t('tooltip.add')}>
+              <AddIcon />
+            </Tooltip>
+          </Fade>
+        </IconButton>
```
Comment (carmeltsadok) [inline src/components/shared/users-input-fields/UsersInputFields.jsx:124]:
```suggestion
           <Fade
            in
            mountOnEnter
            unmountOnExit
            exit={false}
            timeout={500}
          >
            {isModeSearch
              ? (
                <Tooltip title={t('tooltip.search')}>
                  <SearchIcon />
                </Tooltip>
              )
              : (
                <Tooltip title={t('tooltip.add')}>
                  <AddIcon />
                </Tooltip>
              )}
          </Fade>
```

File: src/components/shared/users-input-fields/UsersInputFields.styles.js [hunk 3]
```diff
@@ -26,6 +27,22 @@ const useStyles = makeStyles((theme) => ({
     textAlign: 'center',
     userSelect: 'none',
   },
+  inputArea: {
+    position: 'absolute',
+    top: '0',
+    display: 'flex',
+    flexDirection: 'row',
```
Comment (carmeltsadok) [inline src/components/shared/users-input-fields/UsersInputFields.styles.js:34]: its row by default

File: src/components/shared/users-input-fields/UsersInputFields.jsx [hunk 4]
```diff
@@ -60,7 +70,7 @@ const UsersInputFields = ({
   );
 
   const renderUsersFields = () => (
-    sortedUsers.length > 0 ? sortedUsers.map(({ user, role }) => (
+    updatedUsersList.length > 0 ? sortedUsers.map(({ user, role }) => (
```
Comment (carmeltsadok) [inline src/components/shared/users-input-fields/UsersInputFields.jsx:73]: We need to change the `message.noMembersFound` when there are users in the group, but just there are no search results.

File: src/components/shared/users-input-fields/UsersInputFields.jsx (hunk 2 above)
Comment (carmeltsadok) [reply src/components/shared/users-input-fields/UsersInputFields.jsx:124]: And you haven't changed the order of the buttons :)

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/components/shared/users-input-fields/UsersInputFields.jsx (hunk 2 above)
Comment (PCOffline) [reply src/components/shared/users-input-fields/UsersInputFields.jsx:124]: What do you mean?

File: src/components/shared/users-input-fields/UsersInputFields.jsx (hunk 4 above)
Comment (carmeltsadok) [reply src/components/shared/users-input-fields/UsersInputFields.jsx:73]: Have you done it?

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

**Pull request merged**